.envrc

```diff
@@ -1 +0,0 @@
-use flake . --extra-deprecated-features url-literals
```

Comment:
Did you remove this by accident?

Reply:
No, it's very specific to my setup and I want to store environment variables for development runs there.